### PR TITLE
Implement Java 8 Constants

### PR DESCRIPTION
**Pull request by MylesIsCool**
As per https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html

Untested for writing but tested for reading.

(No automated tests at the moment)

### CODE REVIEW

File: jawa/constants.py [hunk 1]
```diff
@@ -506,7 +575,26 @@ def pack(self, fout):
                     constant._name_index,
                     constant._descriptor_index
                 ))
-
+            elif isinstance(constant, ConstantMethodHandle):
+                            write(pack(
```
Comment (TkTech) [inline jawa/constants.py]: Github messing up here or is this indentation way off?

Comment (MylesIsCool) [inline jawa/constants.py]: It looks fine in raw on GitHub, I've noticed there was a blank line missing after the else if block so I've added that on. Let me know if anything else is missing.

**Comment by TkTech** [comment]
This is what I'm seeing:

image

Just gotta fix the indentation and this can go in, thanks!

**Comment by MylesIsCool** [comment]
@TkTech Formatting should be fixed now

**Pull request merged**